src/pyxtp/CMakeLists.txt

```
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 3.12)
+
+project(pyxtp LANGUAGES CXX)
+
+find_package(pybind11 REQUIRED HINTS)
```

```suggestion
find_package(pybind11 REQUIRED)
```

`HINTS` without a path is pointless.